overlord/devicestate/devicestate_systems_test.go

```diff
@@ -1296,6 +1298,168 @@ func (s *deviceMgrSystemsSuite) TestRecordSeededSystem(c *C) {
 	c.Assert(seededSystemsFromState, DeepEquals, expectedWithNewModel)
 }
 
+func (s *deviceMgrSystemsSuite) mockSystemUser(c *C, username string, expiration time.Time) {
```

Comment:
this is not the right test file for this, as it's not about systems functionality which relates to seed systems

Reply:
I moved them to devicestate_test.go instead and changed the tests a bit.